checker/src/org/checkerframework/checker/index/upperbound/UpperBoundTransfer.java

```diff
@@ -303,14 +303,23 @@ private void refineNeqArrayLength(
     /**
      * If some Node a is known to be less than the length of some array, x, then, the type of a - b,
      * is @LTLengthOf(value="x", offset="b"). If b is known to be less than the length of some other
```

Comment:
Should the offset be `-b`?  I would think that "a+b" is `@LtLengthOf("x", "b")`.